rust/projects/project-3/src/bin/kvs-client.rs

```diff
@@ -4,6 +4,9 @@ use std::net::SocketAddr;
 use std::process::exit;
 use structopt::StructOpt;
 
+const DEFAULT_LISTENING_ADDRESS: &str = "127.0.0.1:4000";
+const ADDRESS_VALUE_NAME: &str = "IP:PORT";
```

Comment:
I think a better name may be `ADDRESS_FORMAT` or `ADDRESS_TEMPLATE`? What do you think?

Reply:
Thanks for your review.  Renamed.